test/testcafe/spec/ChallengeOktaVerifyPush_spec.js

```diff
@@ -58,6 +73,69 @@ test
     await t.expect(challengeOktaVerifyPushPageObject.getSignoutLinkText()).eql('Back to sign in');
   });
 
+test
+  .requestHooks(sendPushMock)('challenge ov push screen has right labels and a checkbox', async t => {
+    const challengeOktaVerifyPushPageObject = await setup(t);
+    await checkConsoleMessages({
+      controller: 'mfa-verify',
+      formName: 'challenge-poll',
+      authenticatorKey: 'okta_verify',
+      methodType: 'push',
+    });
+
+    const pageTitle = challengeOktaVerifyPushPageObject.getFormTitle();
+    const pushBtn = challengeOktaVerifyPushPageObject.getPushButton();
+    const a11ySpan = challengeOktaVerifyPushPageObject.getA11ySpan();
+    const checkboxLabel = challengeOktaVerifyPushPageObject.getAutoChallengeCheckboxLabel(AUTO_CHALLENGE_CHECKBOX_LABEL_SELECTOR);
+    await t.expect(pushBtn.textContent).contains('Push notification sent');
+    await t.expect(a11ySpan.textContent).contains('Push notification sent');
+    await t.expect(pushBtn.hasClass('link-button-disabled')).ok();
+    await t.expect(pageTitle).contains('Get a push notification');
+    await t.expect(await challengeOktaVerifyPushPageObject.autoChallengeInputExists(AUTO_CHALLENGE_CHECKBOX_SELECTOR)).ok();
+    await t.expect(checkboxLabel.hasClass('checked')).ok();
+    await t.expect(checkboxLabel.textContent).eql('Send push automatically');
+
+    // unselect checkbox on click
+    await challengeOktaVerifyPushPageObject.clickAutoChallengeCheckbox(AUTO_CHALLENGE_CHECKBOX_LABEL_SELECTOR);
+    await t.expect(checkboxLabel.hasClass('checked')).notOk();
+
+    // Verify links
+    await t.expect(await challengeOktaVerifyPushPageObject.switchAuthenticatorLinkExists()).ok();
+    await t.expect(challengeOktaVerifyPushPageObject.getSwitchAuthenticatorLinkText()).eql('Verify with something else');
+    await t.expect(await challengeOktaVerifyPushPageObject.signoutLinkExists()).ok();
+    await t.expect(challengeOktaVerifyPushPageObject.getSignoutLinkText()).eql('Back to sign in');
+  });
+
+test
+  .requestHooks(logger, sendPushWaitMock)('should call polling API and checkbox should be clickable after polling started', async t => {
```

Comment:
Earlier you mentioned there is no polling for this view?

Reply:
There are 2 views. Existing one `ChallengePushView` where just the checkbox is added has polling. I was talking about the new view `ChallengeAuthenticatorDataOktaVerifyPushOnlyView` (Send push flow) that has no polling.